mne/epochs.py

```diff
@@ -1914,7 +1914,7 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
                         split_naming, overwrite)
 
     @verbose
-    def export(self, fname, fmt='auto', verbose=None):
+    def export(self, fname, fmt='auto', overwrite=False, verbose=None):
```

Comment:
Easy enough to prevent people from accidentally doing `export(fname, fmt, True)` thinking they're doing `verbose=True)` (plus we actually want all `verbose` to come after a `*` anyway moving forward)
```suggestion
    def export(self, fname, fmt='auto', *, overwrite=False, verbose=None):
```